astropy/coordinates/representation.py

```diff
@@ -436,8 +439,11 @@ def __init__(cls, name, bases, dct):
         repr_name = cls.get_name()
 
         if repr_name in REPRESENTATION_CLASSES:
-            raise ValueError("Representation class {0} already defined"
-                             .format(repr_name))
+            msg = 'Representation {} already defined, using name "{}"'.format(
```

Comment:
```suggestion
            msg = 'Representation {} already defined, storing new representation with name "{}".  Note that this may cause surprising behavior if the representations are actually different!'.format(
```